src/main/resources/mentorshipStudyGroupsPage.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "page:STUDY_GROUPS",
+  "heroSection": {
+    "title": "Technical Study Groups",
+    "images": [
+      {
+        "path": "",
+        "alt": "Image of code on a screen",
+        "type": "desktop"
+      }
+    ]
+  },
+  "section": {
+    "description": "Explore our technical study groups, created in collaboration with available mentors, to deepen your knowledge and skills in specific areas of interest.\n Our study groups cover a variety of topics and are designed to accommodate different schedules and experience levels. Whether you're a beginner looking to dive into a new technology or an experienced professional seeking to expand your expertise, there's a study group for you.\n These study groups typically last anywhere from 1-2 months to longer durations, allowing participants to delve deeply into the subject matter and benefit from the guidance of experienced mentors."
+  },
+  "contact": {
+    "title": "Join us in our Study Group Slack Channel",
+    "links": [
+      {
+        "type": "slack",
+        "link": "https://womencodingcommunity.slack.com/archives/C073W920RND"
+      }
+    ]
+  },
+  "studyGroupSection": {
```

Comment:
this will be changed to be something like this: 

```
"items": [ 
     {
        "title": "Data Analytics and Generative AI concepts",
        "description": "The study group focuses more on the Data Analytics and Generative AI concepts and guiding on the insights across the span of industries upskilling on relevant knowledge and practical work.",
        "coordinartors": "Kirthikka Devi Venkataram",
        "participants":  3,
        "link": {
              "label": "Check her profile",
              "uri": "/mentors?keywords=Kirthikka%20Devi"
          }
     }
]
```